numpy/_core/src/multiarray/multiarraymodule.h

```diff
@@ -21,4 +21,9 @@ NPY_VISIBILITY_HIDDEN extern PyObject * npy_ma_str_convert_if_no_array;
 NPY_VISIBILITY_HIDDEN extern PyObject * npy_ma_str_cpu;
 NPY_VISIBILITY_HIDDEN extern PyObject * npy_ma_str_array_err_msg_substr;
 
+NPY_NO_EXPORT npy_intp
```

Comment:
Is this the right module to define it in? I know `PyArray_EquivTypes` is here, but maybe `casts.c` makes more sense?